Corale.Colore/Core/Headset.cs

```diff
@@ -66,6 +66,7 @@ private Headset()
         /// <param name="color">Color to set.</param>
         public override void SetAll(Color color)
         {
+            Clear();
```

Comment:
Unless @Sharparam agrees with this change, based on the comments in the referenced issue, could we remove this?